src/LocalClasses.java

```diff
@@ -0,0 +1,61 @@
+public class LocalClasses {
+
+    static String regularExpression = "[^0-9]";
+
+    public static void validatePhoneNumber(
+            String phoneNumber1, String phoneNumber2) {
+
+         int numberLength = 10;
+
+        // Valid in JDK 8 and later:
+
+        // int numberLength = 10;
+
+        class PhoneNumber {
+
+            String formattedPhoneNumber = null;
+
+            PhoneNumber(String phoneNumber){
+//                numberLength = 7;
+                String currentNumber = phoneNumber.replaceAll(
```

Comment:
Is there not a better way to verify the input against REGEX?. https://stackoverflow.com/questions/16005234/java-using-regex-to-verify-an-input-string